### PR TITLE
Add region tags to bigtable_hello_world

### PR DESCRIPTION
**Pull request by NettyH**
Added region tags to bigtable_hello_world for GCS.

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-cpp/1544)

### CODE REVIEW

**Comment by coryan** [comment]
Oh yes, the builds are broken.  I will send a fix ASAP.  My bad.

**Comment by coryan** [comment]
Okay, the builds should be fine now, but you will need to resync your `master` branch and rebase your PR to pick up the fixes.  Let me know if you need help with those steps.

**Comment by NettyH** [comment]
Carlos, here are some examples from other languages.
[C#](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/blob/master/bigtable/api/HelloWorld/HelloWorld.cs)
[Nodejs](https://github.com/googleapis/nodejs-bigtable/blob/master/samples/hello-world/index.js)

Thank you

File: google/cloud/bigtable/examples/bigtable_hello_world.cc [hunk 1]
```diff
@@ -80,10 +80,10 @@ int main(int argc, char* argv[]) try {
...
   }
-  //! [write rows]
+  //! [write rows] [END writing_rows]
 
   // Read a single row.
-  //! [read row]
+  //! [read row] [START getting_a_row]
   auto result = table.ReadRow(
       "key-0",
       google::cloud::bigtable::Filter::ColumnRangeClosed("family", "c0", "c0"));
```
Comment (coryan) [inline google/cloud/bigtable/examples/bigtable_hello_world.cc]: The examples in other language separate the filter creation to a separate region tag.  Would it be worth it to move this line to its own line so we can tag it?

File: google/cloud/bigtable/examples/bigtable_hello_world.cc [hunk 2]
```diff
@@ -96,10 +96,10 @@ int main(int argc, char* argv[]) try {
   std::cout << cell.family_name() << ":" << cell.column_qualifier() << "    @ "
             << cell.timestamp().count() << "us\n"
             << '"' << cell.value() << '"' << std::endl;
-  //! [read row]
+  //! [read row] [END getting_a_row]
 
-  // Read a single row.
-  //! [scan all]
+  // Read all rows.
```
Comment (coryan) [inline google/cloud/bigtable/examples/bigtable_hello_world.cc]: Thanks!

**Comment by coryan** [comment]
@NettyH The tests pass and the code looks right.  I think we should merge this and make the changes about the filter tag in a separate PR, but if you already started on those changes we can wait.

**Pull request merged**